dotnetv3/Glue/Tests/GlueTests.cs

```diff
@@ -0,0 +1,238 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using Amazon.Glue;
+using Amazon.Glue.Model;
+using GlueActions;
+using Microsoft.Extensions.Configuration;
+
+namespace GlueTests
+{
+    public class GlueTests
+    {
+        private readonly IConfiguration _configuration;
+        private readonly GlueWrapper _wrapper;
+
+        private readonly IAmazonGlue _glueClient;
+        private string _bucketName;
+        private string _crawlerName;
+        private string _roleName;
+        private string _sourceData;
+        private string _dbName;
+        private string _cron;
+        private string _scriptUrl;
+        private string _jobName;
+        private string _description;
+        private static string _jobRunId;
+        private static List<Table> _tables;
+
+        /// <summary>
+        /// Constructor for the test class.
+        /// </summary>
+        public GlueTests()
+        {
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from .json file.
+                .AddJsonFile("testsettings.local.json",
+                    true) // Optionally load local settings.
+                .Build();
+
+            _glueClient = new AmazonGlueClient();
+            _wrapper = new GlueWrapper(_glueClient);
+
+            _bucketName = _configuration["BucketName"];
```

Comment:
Is _bucketName used?

Reply:
It is used to show what data Glue puts in the bucket.

Reply:
It is only used in the scenario. So I've removed the assignment from the test project.